src/ray/gcs/subscription_executor.cc

```diff
@@ -132,7 +127,8 @@ Status SubscriptionExecutor<ID, Data, Table>::AsyncUnsubscribe(
   return table_.CancelNotifications(JobID::Nil(), id, client_id, on_done);
 }
 
-template class SubscriptionExecutor<ActorID, ActorTableData, ActorTable>;
+template class SubscriptionExecutor<ActorID, std::vector<ActorTableData>, ActorTable>;
+template class SubscriptionExecutor<JobID, std::vector<JobTableData>, JobTable>;
```

Comment:
simple question: why do we need the history for job table data (and thus requires a vector instead of a single entry)?

Reply:
I miss understand the method `NodeManager::HandleJobTableUpdate`, this method will process every record of this job vector.  So if is ok for this method just got the last record of this vector?